storage/mongo/storage_test.go

```diff
@@ -0,0 +1,374 @@
+package mongo
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/netlify/netlify-auth/conf"
+	"github.com/netlify/netlify-auth/models"
+)
+
+var conn *Connection
+
+func TestMain(m *testing.M) {
+	connURL := os.Getenv("NETLIFY_AUTH_MONGODB_TEST_CONN_URL")
+
+	if connURL == "" {
+		fmt.Println(`MongoDB test suite disabled.
+Set the environment variable NETLIFY_AUTH_MONGODB_TEST_CONN_URL with the connection URL to enable them.`)
+		return
+	}
+
+	config := &conf.Configuration{
+		DB: conf.DBConfiguration{
+			Namespace: "netlify_auth",
+			Driver:    "mongodb",
+			ConnURL:   connURL,
+		},
+		JWT: conf.JWTConfiguration{
+			AdminGroupName: "admin-test",
+		},
+	}
+
+	var err error
+	conn, err = Connect(config)
+	if err != nil {
+		panic(err)
+	}
+
+	os.Exit(m.Run())
+}
+
+func TestCreateFirstUser(t *testing.T) {
+	defer cleanTables()
```

Comment:
I've seen that in some of the test suites the using a cleanup method in a `defer` can cause some wonkiness - though it usually works. It is because the defer method is not guaranteed to execute before the next method starts. _usually_ this hasn't been a problem, but I tend to do the cleanup before I start the method instead of after. YMMV